src/api.ts

```diff
@@ -13,14 +13,15 @@ import { l } from './logger';
 import { pendingHashes } from './builder';
 import { exec } from 'child_process';
 
-import { CONTAINER_EXPIRY_TIME, START_TIME, TEN_MINUTES } from './constants';
+import { CONTAINER_EXPIRY_TIME } from './constants';
 import { timing } from './stats';
 
 type APIState = {
-	accesses: Map< CommitHash, number >;
+	accesses: Map< ContainerName, number >;
```

Comment:
This map was only used in this file, by the functions `touchCommit` and `getCommitAccessTime`. Those has been refactored to understand the new data structure.